src/utils/utils.cairo

```diff
@@ -83,6 +103,10 @@ namespace Helpers {
         return fill(arr + 1, value, length - 1);
     }
 
+    // @notice This function fills an empty array with elements from another array
+    // @param fill_with: number of elements to add
+    // @param input_arr: pointer to the input array
+    // @param output_arr: pointer to empty array to be filled with elts from input array
```

Comment:
let's write elements and not elts 
as a general rule, let's avoid short names in comments to help clarity